website/docs/reference/resource-configs/lookback.md

```diff
@@ -3,14 +3,14 @@ title: "lookback"
 id: "lookback"
 sidebar_label: "lookback"
 resource_types: [models]
-description: "dbt uses `lookback` to detrmine how many 'batches' of `batch_size` to reprocesses when a microbatch incremental model is running incrementally."
+description: "dbt uses `lookback` to determine how many 'batches' of `batch_size` to reprocess when a microbatch incremental model is running incrementally."
```

Comment:
Sort of saying the same thing twice at the end there
```suggestion
description: "Configure `lookback` to determine how many 'batches' of `batch_size` to reprocess when a dbt microbatch incremental model is running."
```

Reply:
it is you're right! but my point of view is that not everyone may immediately grasp that an incremental model is, by definition, running incrementally. based on my experience with support, i like to try to be explicit as possible . also didn't think it was a big risk since this is in the description? how about:

> Configure lookback to determine how many 'batches' of batch_size to reprocess when a dbt microbatch incremental model runs incrementally